catalog-info.yaml

```diff
@@ -212,3 +212,42 @@ spec:
 # ***********************************
 # SECTION END: DRA pipelines
 # ***********************************
+
+# ********************************************
+# Declare supported plugin tests pipeline
+# ********************************************
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: 'Logstash supported plugins test pipeline'
+  description: "Execute spec tests af all supported tier1 and tier2 plugins using the current branch's Logstash"
+  links:
+    - title: 'Logstash supported plugins test pipeline'
+      url: https://buildkite.com/elastic/logstash-supported-plugins-test-pipeline
+spec:
+  type: buildkite-pipeline
+  owner: group:logstash
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: 'Logstash supported plugins test pipeline'
```

Comment:
Double checked, this will work fine and is used e.g. in [elasticsearch](https://github.com/elastic/elasticsearch/blob/d214d74000b57a4cd121a0b4922ce5acc48edf73/catalog-info.yaml#L252).
The resulting URL will replace spaces with dashes e.g. the elasticsearch one becomes https://buildkite.com/elastic/elasticsearch-ecs-dynamic-templates-tests

Reply:
Ok, thanks for checking. I saw that elasticsearch uses a sort of layered naming convention like:
```
<product> / <area> / <task>
```

Do you think we should follow the same convention or stick with a more "phrasal naming" like it's now, or something else?

Reply:
Naming is hard! I think it's matter of preference. What you have here looks good to me. In Elasticsearch, due to the project size, there's probably a bigger need to organize things like that, and I think it's also legacy from how it was done in the past on Jenkins. Main thing for me is to have something that is readable and understandable even when glancing at the `name` and not consulting the `description`.